hera_mc/autocorrelations.py

```diff
@@ -31,46 +33,97 @@ class _MeasurementTypes(object):
     median = 0
     "The median value across the whole autocorrelation spectrum."
 
-    names = ['median']
+    names = ["median"]
     "A list of textual names corresponding to each value."
 
 
 MeasurementTypes = _MeasurementTypes()
 
 
-class Autocorrelations(MCDeclarativeBase):
-    """A table logging antenna autocorrelations.
-
+class HeraAuto(MCDeclarativeBase):
+    """
+    Definition of median antenna autocorrelation table of hera antennas.
+
+    Attributes
+    ----------
+    time : BigInteger Column
+        The time in GPS seconds of the observation, floored as an int. Part of primary_key
+    antenna_number : Integer Column
+        Antenna number. Part of primary_key.
+    antenna_feed_pol : String Column
+        Feed polarization, either 'e' or 'n'. Part of primary_key.
+    measurement_type : SmallInt Column
+        The type of measurement; see MeasurementTypes enumeration.
+        Cannot be None.
+    value : Float Columnn
+        53 precision autocorrelation value.
+        Cannot be None
     """
-    __tablename__ = 'autocorrelations'
-
-    id = Column(BigInteger, primary_key=True)
-    "A unique ID number for each record; no intrinsic meaning."
-
-    time = NotNull(DateTime)
-    "The time when the information was generated; stored as SqlAlchemy UTC DateTime."
 
-    antnum = NotNull(Integer)
-    "The internal antenna number to which this record pertains."
+    __tablename__ = "hera_autos"
 
-    polarization = NotNull(String(1))
-    """Which polarization this record refers to: "x" or "y"."""
+    time = Column(BigInteger, primary_key=True)
+    antenna_number = Column(Integer, primary_key=True)
+    antenna_feed_pol = Column(String, primary_key=True)
+    measurement_type = Column(SmallInteger, nullable=False)
+    # recommended portable way of getting double-precision float.
+    value = Column(Float(precision="53"), nullable=False)
```

Comment:
It turns out that Float is mapped to double precision in postgres (which is the db we'll be using exclusively for this table) so I don't think this is required.